### ISSUE TITLE
Unfinished "Partials & Layouts" documentation

### ISSUE THREAD

**Issue by Arturace** [squirrellyjs/squirrelly-docs#14]
The documentation for [Partials & Layouts](https://squirrelly.js.org/docs/api/templates-partials-layouts) is unfinished. No mention of `extends` and `extendsFile`

**Comment by bgub** [comment]
You're right! I completely forgot to put anything about layouts in the documentation.

I'll begin working on that right now :)

**Comment by Arturace** [comment]
Thank you! I've been enjoying your framework a lot so far.
